externs/browser/window.js

```diff
@@ -134,15 +134,15 @@ function alert(message) {}
 function clearImmediate(immediateID) {}
 
 /**
- * @param {number|undefined?} intervalID
+ * @param {number|?undefined} intervalID
```

Comment:
`?undefined` should just be `undefined`.

Reply:
I was under the impression that this would change the semantics. i.e. `?undefined` allowed null while `undefined` just allowed the omission of parameter. However looking at the spec for [WindowOrWorkerGlobalScope](https://html.spec.whatwg.org/multipage/webappapis.html#windoworworkerglobalscope) it seems that it is probably better represented by `@param {number=} opt_intervalID` ... one moment

Reply:
Oops, you're right about `?undefined`. Thanks for noticing it's optional.